src/background/background.js

```
@@ -52,33 +52,37 @@ chrome.commands.onCommand.addListener((command) => {
       });
+    } else {
+      console.log("Tab or tab.url is undefined");
     }
   }
 });
 
 // Also handle initial tab loading
 chrome.tabs.onActivated.addListener((activeInfo) => {
   chrome.tabs.get(activeInfo.tabId, (tab) => {
-    if (tab.url.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
+    if (tab && tab.url && tab.url.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)